state/megawatcher.go

```diff
@@ -491,6 +496,86 @@ func backingEntityIdForSettingsKey(key string) (eid multiwatcher.EntityId, extra
 	return
 }
 
+type backingOpenedPorts map[string]interface{}
+
+func (p *backingOpenedPorts) updated(st *State, store *multiwatcherStore, id interface{}) error {
+	localID := st.localID(id.(string))
+	parentId, ok := backingEntityIdForOpenedPortsKey(localID)
+	if !ok {
+		return nil
+	}
+	info0 := store.Get(parentId)
+	switch info := info0.(type) {
```

Comment:
just:

     switch info := store.Get(parentId).(type) {

should be fine, I think.